src/compute/src/logging/reachability.rs

```diff
@@ -77,36 +81,44 @@ pub(super) fn construct<A: Allocate>(
         use timely::dataflow::channels::pact::Pipeline;
         let mut input = flatten.new_input(&logs, Pipeline);
 
-        let (mut updates_out, updates) = flatten.new_output::<ConsolidatingContainerBuilder<_>>();
```

Comment:
I'll do some measurements to confirm that removing the eager consolidation is not a regression.